src/app/layout/footer/footer.component.scss

```diff
@@ -0,0 +1,5 @@
+@import '../../../styles.colors.scss';
+
+.page-footer {
+    background-color: $primary-color;
```

Comment:
Use 2 spaces for indentation as stated in the `.editorconfig`

Reply:
Thank you, fixed.